srcpkgs/systemd-boot/template

```diff
@@ -0,0 +1,58 @@
+# Template file for 'systemd-boot'
+pkgname=systemd-boot
+version=241
+revision=1
+archs="~*-musl"
+wrksrc="systemd-${version}"
+build_style=meson
+configure_args="-Dblkid=true -Defi=true -Dgnu-efi=true
+ -Defi-cc=$CC -Dsplit-usr=false -Drootprefix=/
+ -Dacl=false -Dapparmor=false -Daudit=false -Dbzip2=false
+ -Delfutils=false -Dgcrypt=false -Dgnutls=false -Dkmod=false
+ -Dlibcryptsetup=false -Dlibcurl=false -Dlibidn=false
+ -Dlibidn2=false -Dlibiptc=false -Dlz4=false -Dmicrohttpd=false
+ -Dmyhostname=false -Dpam=false -Dqrencode=false -Dseccomp=false
+ -Dselinux=false -Dxkbcommon=false -Dxz=false
+ -Defi-includedir=${XBPS_CROSS_BASE}/usr/include/efi
+ -Defi-ldsdir=${XBPS_CROSS_BASE}/usr/lib -Defi-ld=${XBPS_CROSS_BASE}/usr/bin/ld
+ -Defi-libdir=${XBPS_CROSS_BASE}/usr/lib"
+hostmakedepends="gperf patchelf pkg-config libxslt intltool m4 docbook-xsl"
+makedepends="libcap-devel gnu-efi-libs libblkid-devel libmount-devel"
+short_desc="UEFI boot manager from systemd"
+maintainer="maxice8 <thinkabit.ukim@gmail.com>"
+license="LGPL-2.1-or-later"
+homepage="https://www.freedesktop.org/wiki/Software/systemd/systemd-boot"
+distfiles="https://github.com/systemd/systemd/archive/v${version}.tar.gz"
+checksum=b2561a8e1d10a2c248253f0dda31a85dd6d69f2b54177de55e02cd1d2778316e
+
+case "$XBPS_TARGET_MACHINE" in
+	x86_64*) efi_arch=x64 ;;
+	armv*) efi_arch=arm ;;
+	aarch64*) efi_arch=aa64 ;;
+	i686*) efi_arch=x86 ;;
```

Comment:
add a `*) broken="efi_arch is not set";;` Or does it have a generic `src/boot/efi/linux.efi.stub` target for ninja?

Reply:
Just use the wrong name for the i686 arch mostlikely

Reply:
I don't think `broken` is right here, I would prefer `archs=` as unsupported platforms not the same as broken platforms.